schema.sql

```diff
@@ -1,5 +1,3 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE ANIMALS (ID INTEGER, NAME VARCHAR(30), DATE_OF_BIRTH DATE, ESCAPE_ATTEMPTS INTEGER, NEUTERED BOOLEAN, WEIGHT_KG Decimal)
```

Comment:
[OPTIONAL]
- [ ] I would kindly suggest you make the `id` column not null and autoincremented. Also i will advice you mark the id column as primary key

E.g
```JS
    id int NOT NULL IDENTITY(1,1)
```